projects/canopy/src/lib/pagination/docs/guide.stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import Feedback from '../../docs/common/feedback.md';
+
+<Meta title="Components/Pagination/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
```

Comment:
I would change the title to be `Components/Pagination (WIP)/Guide` (and for the examples too).

Reply:
done